src/components/SlateEditor/plugins/related/EditRelated.jsx

```diff
@@ -165,7 +227,11 @@ class EditRelated extends React.PureComponent {
                   {t('form.content.relatedArticle.addExternal')}
                 </Button>
               </StyledArticle>
-              <DeleteButton stripped onClick={onRemoveClick} />
+              <DeleteButton
+                stripped
+                onClick={onRemoveClick}
+                aria-label={t('topicArticleForm.removeVisualElement')}
```

Comment:
Fint med aria-label men teksten som vises, er den korrekt?

Reply:
får ikke til å endre teksten som vises uten å endre på komponenten tror jeg. Tenkte å hive på en aria-label som et plaster selv om det ikke hjelper for de seende